package/MDAnalysis/core/groups.py

```diff
@@ -1044,7 +1044,7 @@ def forces(self, values):
         ts = self._u.trajectory.ts
         try:
             ts.forces[self._ix, :] = values
-        except AttributeError:
+        except (AttributeError, NoDataError):
```

Comment:
Timestep always threw a NoDataError, so this wasn't caught by AG before. Now it's caught and a better error message given